os/tools/mkbinheader.py

```diff
@@ -68,34 +68,66 @@
 ELF = 1
 BIN = 2
 
+# In size command on linux, 4th value is the summation of text, data and bss.
+# We will use this value for elf type.
```

Comment:
You use this definition outside of elf case. What is right?

Reply:
And is this valid in esp? Does esp use GNU toolchain?

Reply:
1. Yes. It is okay. Because we always open the STATIC_RAM_ESTIMATION with elf type, even kernel binary.
2. I tested with the esp output file. It works same.